src/utils/constants.ts

```diff
@@ -147,11 +147,10 @@ export default {
   TITLE_HISTORY_REVIEW: 'Review',
   TITLE_HISTORY_SUBMITTED_BY_APPLICANT: 'Submitted by applicant',
   TITLE_HISTORY_PANEL: 'Review History',
-  TITLE_LOGIN_HEADER: 'IRIMS Application Manager',
+  TITLE_LOGIN_HEADER: 'mFlow - Medical Forms, Licensing & Organizational Workflow',
```

Comment:
This seems way too long, needs to fit on one line like the original one does. 

Also -- maybe ditch this string and just use the exising APP_NAME and APP_NAME_SUBHEADER from the top of the file?

Reply:
Also -- "MFLOW" looks quite bad in all caps. The "m" should probable _never_ be capitalised (like the "i" in iPhone* or iPad), so it's only ever "mFlow", or (rarely) "mFLOW".  So you'll probably need to remove the "text-transform" prop on "login-header" -- not sure if we'd want the rest capitalised or not here, have a play and see what works best.

Guess that's the kind of thing that needs to go into a style guide.

\* If you ever write IPHONE, Steve Jobs will emerge from the grave and eat your soul... shit, what's tha...